src/pages/AdminPage.jsx

```diff
@@ -0,0 +1,457 @@
+import { React, useState, useEffect } from 'react';
+import {
+  Th,
+  Thead,
+  Table,
+  Td,
+  Tr,
+  Tbody,
+  Heading,
+  Checkbox,
+  Select,
+  InputGroup,
+  InputRightAddon,
+  Input,
+  Container,
+  Text,
+  Flex,
+  IconButton,
+  Spacer,
+  HStack,
+  Box,
+  Button,
+} from '@chakra-ui/react';
+import { ChevronRightIcon, ChevronLeftIcon } from '@chakra-ui/icons';
+import DatePicker from 'react-datepicker';
+import './AdminPage.css';
+
+const dummy = [
+  {
+    id: 0,
+    segment: 'OC21',
+    date: '11/15/2021',
+    approved: 'APPROVED',
+    volunteer: 'Segun Adebayo',
+    status: 'IN-TRAINING',
```

Comment:
Training status should be a boolean value, since that's how the submission schema is written. Can you change this as well as anywhere else you processing training status